.github/workflows/shellcheck-posix.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       SHELLCHECK_VERSION: '0.9.0'
-      SHELLCHECK_OPTS: '-e SC2009'
+      SHELLCHECK_OPTS: '-e SC1091 -e SC2009'
```

Comment:
I would like to add description what this numbers means - what rule is exclude

Reply:
Done.